src/main/drivers/at32/system_at32f43x.c

```diff
@@ -29,6 +29,40 @@
 #include "drivers/persistent.h"
 #include "at32f435_437_clock.h"
 
+static flash_usd_eopb0_type get_sram_config(void)
+{
+    extern uint32_t _SRAM_SIZE; // Defined in linker file
+    switch ((uint32_t)&_SRAM_SIZE) {
+    case 128:
+        return FLASH_EOPB0_SRAM_128K;
+    case 256:
+        return FLASH_EOPB0_SRAM_256K;
+    case 320:
+        return FLASH_EOPB0_SRAM_320K;
+    case 384:
+        return FLASH_EOPB0_SRAM_384K;
+    case 448:
+        return FLASH_EOPB0_SRAM_448K;
+    case 512:
+        return FLASH_EOPB0_SRAM_512K;
+    case 192:
+    default:
+        return FLASH_EOPB0_SRAM_192K;
+    }
+}
+
+static void init_sram_config(void)
+{
+    // Make sure the SRAM config is correct
+    const flash_usd_eopb0_type sram_cfg = get_sram_config();
+    if (((USD->eopb0) & 0x07) != sram_cfg) {
```

Comment:
Define a constant here. What does 0x07 mean?

Reply:
Good point! Unfortunately at32 lib does not contain define for a mask. Will add 👍

Reply:
fixed.